### ISSUE TITLE
[BUG] Error when Oban Plugins are configured with options

### ISSUE THREAD

**Issue by moogle19** [akoutmos/prom_ex#34]
**Describe the bug**

When Oban plugins (e.g. `Oban.Pruner`) are configured with options, `PromEx` fails during startup.
I suspect it's because the Oban plugins are converted to string, but this fails when options for the plugin are provided.
Following error occurs:

```elixir
[error] Handler {PromEx.TelemetryMetricsPrometheus.Core.EventHandler, #PID<0.3165.0>,
 [:my_app, :prom_ex, :oban, :init, :status, :info]} has failed and has been detached. Class=:error
Reason=%Protocol.UndefinedError{
  description: "",
  protocol: String.Chars,
  value: {Oban.Plugins.Pruner, [max_age: 2592000]}
}
Stacktrace=[
  {String.Chars, :impl_for!, 1, [file: 'lib/string/chars.ex', line: 3]},
  {String.Chars, :to_string, 1, [file: 'lib/string/chars.ex', line: 22]},
  {Enum, :"-join/2-lists^foldl/2-0-", 3, [file: 'lib/enum.ex', line: 2193]},
  {Enum, :join, 2, [file: 'lib/enum.ex', line: 2193]},
  {PromEx.Plugins.Oban, :oban_init_tag_values, 1,
   [file: 'lib/prom_ex/plugins/oban.ex', line: 476]},
  {PromEx.TelemetryMetricsPrometheus.Core.LastValue, :handle_event, 4,
   [file: 'lib/core/last_value.ex', line: 57]},
  {:telemetry, :"-execute/3-fun-0-", 4,
   [
     file: '/Path/To/Project/my_app/deps/telemetry/src/telemetry.erl',
     line: 135
   ]},
  {:lists, :foreach, 2, [file: 'lists.erl', line: 1342]},
  {Oban, :init, 1, [file: 'lib/oban.ex', line: 183]},
  {:supervisor, :init, 1, [file: 'supervisor.erl', line: 301]},
  {:gen_server, :init_it, 2, [file: 'gen_server.erl', line: 417]},
  {:gen_server, :init_it, 6, [file: 'gen_server.erl', line: 385]},
  {:proc_lib, :init_p_do_apply, 3, [file: 'proc_lib.erl', line: 226]}
]

[error] Handler {PromEx.TelemetryMetricsPrometheus.Core.EventHandler, #PID<0.3165.0>,
 [:my_app, :prom_ex, :oban, :init, :circuit, :backoff, :milliseconds]} has failed and has been detached. Class=:error
Reason=%Protocol.UndefinedError{
  description: "",
  protocol: String.Chars,
  value: {Oban.Plugins.Pruner, [max_age: 2592000]}
}
Stacktrace=[
  {String.Chars, :impl_for!, 1, [file: 'lib/string/chars.ex', line: 3]},
  {String.Chars, :to_string, 1, [file: 'lib/string/chars.ex', line: 22]},
  {Enum, :"-join/2-lists^foldl/2-0-", 3, [file: 'lib/enum.ex', line: 2193]},
  {Enum, :join, 2, [file: 'lib/enum.ex', line: 2193]},
  {PromEx.Plugins.Oban, :oban_init_tag_values, 1,
   [file: 'lib/prom_ex/plugins/oban.ex', line: 476]},
  {PromEx.TelemetryMetricsPrometheus.Core.LastValue, :handle_event, 4,
   [file: 'lib/core/last_value.ex', line: 57]},
  {:telemetry, :"-execute/3-fun-0-", 4,
   [
     file: '/Path/To/Project/my_app/deps/telemetry/src/telemetry.erl',
     line: 135
   ]},
  {:lists, :foreach, 2, [file: 'lists.erl', line: 1342]},
  {Oban, :init, 1, [file: 'lib/oban.ex', line: 183]},
  {:supervisor, :init, 1, [file: 'supervisor.erl', line: 301]},
  {:gen_server, :init_it, 2, [file: 'gen_server.erl', line: 417]},
  {:gen_server, :init_it, 6, [file: 'gen_server.erl', line: 385]},
  {:proc_lib, :init_p_do_apply, 3, [file: 'proc_lib.erl', line: 226]}
]

[error] Handler {PromEx.TelemetryMetricsPrometheus.Core.EventHandler, #PID<0.3165.0>,
 [:my_app, :prom_ex, :oban, :init, :shutdown, :grace, :period, :milliseconds]} has failed and has been detached. Class=:error
Reason=%Protocol.UndefinedError{
  description: "",
  protocol: String.Chars,
  value: {Oban.Plugins.Pruner, [max_age: 2592000]}
}
Stacktrace=[
  {String.Chars, :impl_for!, 1, [file: 'lib/string/chars.ex', line: 3]},
  {String.Chars, :to_string, 1, [file: 'lib/string/chars.ex', line: 22]},
  {Enum, :"-join/2-lists^foldl/2-0-", 3, [file: 'lib/enum.ex', line: 2193]},
  {Enum, :join, 2, [file: 'lib/enum.ex', line: 2193]},
  {PromEx.Plugins.Oban, :oban_init_tag_values, 1, 
   [file: 'lib/prom_ex/plugins/oban.ex', line: 476]},
  {PromEx.TelemetryMetricsPrometheus.Core.LastValue, :handle_event, 4,
   [file: 'lib/core/last_value.ex', line: 57]},
  {:telemetry, :"-execute/3-fun-0-", 4,
   [
     file: '/Path/To/Project/my_app/deps/telemetry/src/telemetry.erl',
     line: 135
   ]},
  {:lists, :foreach, 2, [file: 'lists.erl', line: 1342]},
  {Oban, :init, 1, [file: 'lib/oban.ex', line: 183]},
  {:supervisor, :init, 1, [file: 'supervisor.erl', line: 301]},
  {:gen_server, :init_it, 2, [file: 'gen_server.erl', line: 417]},
  {:gen_server, :init_it, 6, [file: 'gen_server.erl', line: 385]},
  {:proc_lib, :init_p_do_apply, 3, [file: 'proc_lib.erl', line: 226]}
]

[error] Handler {PromEx.TelemetryMetricsPrometheus.Core.EventHandler, #PID<0.3165.0>,
 [:my_app, :prom_ex, :oban, :init, :dispatch, :cooldown, :milliseconds]} has failed and has been detached. Class=:error
Reason=%Protocol.UndefinedError{
  description: "",
  protocol: String.Chars,
  value: {Oban.Plugins.Pruner, [max_age: 2592000]}
}
Stacktrace=[
  {String.Chars, :impl_for!, 1, [file: 'lib/string/chars.ex', line: 3]},
  {String.Chars, :to_string, 1, [file: 'lib/string/chars.ex', line: 22]},
  {Enum, :"-join/2-lists^foldl/2-0-", 3, [file: 'lib/enum.ex', line: 2193]},
  {Enum, :join, 2, [file: 'lib/enum.ex', line: 2193]},
  {PromEx.Plugins.Oban, :oban_init_tag_values, 1,
   [file: 'lib/prom_ex/plugins/oban.ex', line: 476]},
  {PromEx.TelemetryMetricsPrometheus.Core.LastValue, :handle_event, 4,
   [file: 'lib/core/last_value.ex', line: 57]},
  {:telemetry, :"-execute/3-fun-0-", 4,
   [
     file: '/Path/To/Project/my_app/deps/telemetry/src/telemetry.erl',
     line: 135
   ]},
  {:lists, :foreach, 2, [file: 'lists.erl', line: 1342]},
  {Oban, :init, 1, [file: 'lib/oban.ex', line: 183]},
  {:supervisor, :init, 1, [file: 'supervisor.erl', line: 301]},
  {:gen_server, :init_it, 2, [file: 'gen_server.erl', line: 417]},
  {:gen_server, :init_it, 6, [file: 'gen_server.erl', line: 385]},
  {:proc_lib, :init_p_do_apply, 3, [file: 'proc_lib.erl', line: 226]}
]
```

Oban config:

```elixir
config :my_app, Oban,
  repo: MyApp.Repo,
  plugins: [{Oban.Plugins.Pruner, max_age: 2_592_000}],
  queues: [default: 4, events: 4, media: 4]
```

**To Reproduce**

1. Add `Oban` to your application and configure `Oban.Pruner` with a max_age.
2. Add `PromEx.Plugins.Oban`

**Expected behavior**
It shouldn't produce an error.

**Environment**

- Elixir version: 1.11.3
- Erlang/OTP version: 23
- Grafana version: -
- Prometheus version: -

**Comment by akoutmos** [comment]
The fix will be included in 1.0.1. Thanks again!

**Comment by akoutmos** [comment]
Just published 1.0.1 to Hex https://hex.pm/packages/prom_ex
